### PR TITLE
refactor: blockquote

### PR DESCRIPTION
**Pull request by rohitp-bsf**
### Description
- refactor: updated blockquite module.

### Screenshots
- NA

### Types of changes
- Refactored Class components to Functional components.
- Refactored the SCSS files.
- Refactored the import statement.

### How has this been tested?
- NA

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/blockquote/components/AuthorText.js [hunk 1]
```diff
@@ -1,57 +1,50 @@
...
-	                className='uagb-blockquote__author'
-	                onChange={ ( value ) => setAttributes( { author: value } ) }
-	                onMerge = { props.mergeBlocks }
-	                onSplit = {
-						props.insertBlocksAfter ?
-							( before, after, ...blocks ) => {
-								setAttributes( { content: before } )
+const { RichText } = wp.blockEditor;
+
+const { __ } = wp.i18n;
```
Comment (RajkiranBagal) [inline src/blocks/blockquote/components/AuthorText.js:2]: Replace with `import { __ } from '@wordpress/i18n';`

File: src/blocks/blockquote/components/Description.js [hunk 2]
```diff
@@ -1,57 +1,53 @@
...
-	                className='uagb-blockquote__content'
-	                onChange={ ( value ) => setAttributes( { descriptionText: value } ) }
-	                onMerge = { props.mergeBlocks }
-	                onSplit = {
-						props.insertBlocksAfter ?
-							( before, after, ...blocks ) => {
-								setAttributes( { content: before } )
+const { RichText } = wp.blockEditor;
+
+const { __ } = wp.i18n;
```
Comment (RajkiranBagal) [inline src/blocks/blockquote/components/Description.js:2]: Replace with import { __ } from '@wordpress/i18n';

File: src/blocks/blockquote/components/TweetButton.js [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
+import UAGB_Block_Icons from "@Controls/block-icons";
+const TweetButton = ( { attributes } ) => (
+	<a
+		href="javascript:void(0)"
+		className="uagb-blockquote__tweet-button"
+		target="_blank"
+		rel="noopener noreferrer"
+	>
+		{ attributes.iconView === 'icon_text' && (
+			<Fragment>
```
Comment (RajkiranBagal) [inline src/blocks/blockquote/components/TweetButton.js:11]: We can eliminate Fragment from everywhere.

File: src/blocks/blockquote/components/TweetButtonCTA.js [hunk 4]
```diff
@@ -1,31 +1,32 @@
-const { Fragment } = wp.element
-import UAGB_Block_Icons from "@Controls/block-icons"
-class TweetButtonCTA extends React.Component {
+const { Fragment } = wp.element;
```
Comment (RajkiranBagal) [inline src/blocks/blockquote/components/TweetButtonCTA.js:1]: We can eliminate Fragment

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Comment by RajkiranBagal** [comment]
Let's keep PR titles consistent like: 

`refactor: Block Name`

File: src/blocks/blockquote/components/AuthorText.js (hunk 1 above)
Comment (rohitp-bsf) [reply src/blocks/blockquote/components/AuthorText.js]: Done 👍

File: src/blocks/blockquote/components/Description.js (hunk 2 above)
Comment (rohitp-bsf) [reply src/blocks/blockquote/components/Description.js]: Done 👍

File: src/blocks/blockquote/components/TweetButton.js (hunk 3 above)
Comment (rohitp-bsf) [reply src/blocks/blockquote/components/TweetButton.js]: Done 👍

File: src/blocks/blockquote/edit.js [hunk 5]
```diff
@@ -1,30 +1,30 @@
...
-import TypographyControl from "../../components/typography"
+import TypographyControl from '../../components/typography';
 
 // Import Web font loader for google fonts.
-import WebfontLoader from "../../components/typography/fontloader"
-
-import { __ } from '@wordpress/i18n';
+import WebfontLoader from '../../components/typography/fontloader';
 
+const { __ } = wp.i18n;
```
[the remark this answers is not in the document]
Comment (rohitp-bsf) [reply src/blocks/blockquote/edit.js]: Done 👍

File: src/blocks/blockquote/edit.js [hunk 6]
```diff
@@ -38,1141 +38,1829 @@ const {
...
+									'Username',
+									'ultimate-addons-for-gutenberg'
+								) }
+								type="text"
+								value={ iconShareVia }
+							/>
+						</Toolbar>
+					</Fragment>
+				) }
+			</BlockControls>
```
[the remark this answers is not in the document]
Comment (rohitp-bsf) [reply src/blocks/blockquote/edit.js]: Done 👍

File: src/blocks/blockquote/edit.js [hunk 7]
```diff
@@ -38,1141 +38,1829 @@ const {
...
-									) }	>
-					      		{ <AuthorImage attributes={attributes} /> }
-					      		{ <AuthorText attributes={attributes} setAttributes = { setAttributes } props = { this.props } /> }
-									</div>
-					      	{ enableTweet &&  <TweetButtonCTA attributes={attributes} /> }
-					   </footer>
+					className={ classnames(
+						'uagb-blockquote__wrap',
+						`uagb-blockquote__skin-${ skinStyle }`,
+						skinStyle !== 'border'
```
[the remark this answers is not in the document]
Comment (rohitp-bsf) [reply src/blocks/blockquote/edit.js]: Done 👍

**Comment by rohitp-bsf** [comment]
> Let's keep PR titles consistent like:
> 
> `refactor: Block Name`

Yup, Sure! Updated the PR title.

**Review by supriyajjadhav — APPROVED** [review]

File: src/blocks/blockquote/components/AuthorImage.js [hunk 8]
```diff
@@ -1,39 +1,38 @@
...
+const AuthorImage = ( { attributes } ) => {
+	let url_chk = '';
 
-	render() {
+	if (
+		typeof attributes.authorImage !== 'undefined' &&
+		attributes.authorImage !== null &&
+		attributes.authorImage !== ''
+	) {
+		url_chk = attributes.authorImage.url;
```
Comment (RajkiranBagal) [inline src/blocks/blockquote/components/AuthorImage.js:9]: User camelCase variable names.

File: src/blocks/blockquote/edit.js [hunk 9]
```diff
@@ -1,1178 +1,50 @@
...
+		const $style = document.createElement( 'style' );
+		$style.setAttribute(
+			'id',
+			'uagb-blockquote-style-' + props.clientId.substr( 0, 8 )
+		);
+		document.head.appendChild( $style );
+	}, [] );
+
+	return (
+		<>
```
Comment (RajkiranBagal) [inline src/blocks/blockquote/edit.js:41]: Remove extra `<>`

File: src/blocks/blockquote/settings.js [hunk 10]
```diff
@@ -0,0 +1,1753 @@
...
+					aria-pressed={ quotePaddingType === key }
+					onClick={ () => setAttributes( { quotePaddingType: key } ) }
+				>
+					{ name }
+				</Button>
+			) ) }
+		</ButtonGroup>
+	);
+
+	let image_name = __( 'Select Image', 'ultimate-addons-for-gutenberg' );
```
Comment (RajkiranBagal) [inline src/blocks/blockquote/settings.js:1625]: User camelCase variable names

File: src/blocks/blockquote/styling.js [hunk 11]
```diff
@@ -76,237 +75,340 @@ function styling( props ) {
...
+		] = {
+			'border-right-color': tweetBtnBgHoverColor,
+		};
 	}
 
-	var tablet_selectors = {
-		" .uagb-blockquote__content.block-editor-rich-text__editable": {
-			"font-size": generateCSSUnit( descFontSizeTablet, descFontSizeType ),
-			"line-height": generateCSSUnit( descLineHeightTablet, descLineHeightType ),
+	const tablet_selectors = {
```
Comment (RajkiranBagal) [inline src/blocks/blockquote/styling.js:290]: User camelCase variable names throughout the file.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/blockquote/styling.js (hunk 11 above)
Comment (rohitp-bsf) [reply src/blocks/blockquote/styling.js]: Done!

File: src/blocks/blockquote/settings.js (hunk 10 above)
Comment (rohitp-bsf) [reply src/blocks/blockquote/settings.js]: Done!

File: src/blocks/blockquote/edit.js (hunk 9 above)
Comment (rohitp-bsf) [reply src/blocks/blockquote/edit.js]: Done!

File: src/blocks/blockquote/components/AuthorImage.js (hunk 8 above)
Comment (rohitp-bsf) [reply src/blocks/blockquote/components/AuthorImage.js]: Done!

**Review by sandeshjangam — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**